src/apps/relay/dbdrivers/dbd_sqlite.c

```diff
@@ -194,15 +191,12 @@ static void init_sqlite_database(sqlite3 *sqliteconnection) {
       "CREATE TABLE admin_user (name varchar(32), realm varchar(127), password varchar(127), primary key (name))",
       NULL};
 
-  int i = 0;
-  while (statements[i]) {
+  for (int i = 0; statements[i] != NULL; ++i) {
```

Comment:
initialize variables as close to where they are defined as possible.

in this case, by using a for loop instead of while loop.